tracer/src/Datadog.Tracer.Native/rejit_preprocessor.cpp

```diff
@@ -200,6 +200,17 @@ ULONG RejitPreprocessor<RejitRequestDefinition>::RequestRejitForLoadedModules(
     return rejitCount;
 }
 
+template <class RejitRequestDefinition>
```

Comment:
Not an observation on your changes, but looking through the Request and Revert code, so much of the code looks the same except one calls Request methods and one calls Revert methods. It would be amazing if we could unify that code somehow

Reply:
Yes you are right. But it is green 😭
I'll give it a try :)